app/javascript/components/ArticleForm.js

```diff
@@ -0,0 +1,100 @@
+import axios from 'axios';
+import React, { useState } from 'react';
+import { Form, FormGroup, TextInput, Modal, ModalVariant, Button } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+
+const ArticleForm = ({ isOpen, handleModalToggle }) => {
+  const [data, setData] = useState({
+    title: '',
+    description: '',
+    creator: ''
+  });
+
+  const handleChange = (value, key) => {
+    setData({
+      ...data,
+      [key]: value
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const userData = {
+      title: data.title,
+      description: data.description,
```

Comment:
actually `description` is not correct, it should be `body`

Reply:
Yes, that was a mistake. Thanks for the mention :)